### DISCUSSION TITLE
overflow cause blur issue in ipad Air

### DISCUSSION THREAD

**Discussion by kimmingyu1234756** [tailwindlabs/tailwindcss#11702] [Help]
What version of Tailwind CSS are you using?

v.3.2.7

What build tool (or framework if it abstracts the build tool) are you using?

"webpack-cli": "^5.0.1",
"webpack-dev-server": "^4.12.0",
"workbox-webpack-plugin": "^6.5.4"
"react": "^18.2.0",
"react-dom": "^18.2.0",
"react-router-dom": "^6.9.0",

What version of Node.js are you using?

v16.15.1

What browser are you using?

Chrome

What operating system are you using?

Windows10

Describe your issue

Describe the problem you're seeing, any important steps to reproduce and what behavior you expect instead.

```js

 <div className="flex flex-col w-1/2 h-full gap-2 p-2 overflow-y-scroll">
      {selectedMenuCategoryOptions.map((selectedMenuCategoryOption) => (
        <div
          className="flex flex-col w-1/2 border-2 border-black gap-2 p-2"
          key={selectedMenuCategoryOption.categoryId}
        >  </div>
))}
      <div
        className="flex items-center border-2 border-black w-1/2"
        onClick={selectedMenuCategoryOptionsIncrease}
      >
        <PlusIcon className="w-4 h-4" />
        <div>Category</div>
      </div>
    </div>

//onClick function, selectedMenuCategoryOptionsIncrease is functions for creating divs in map()

```

I ran into a problem using overflow. 
I was stretching div tags map in react, but when I used overflow, I was getting a blurry screen.

- before overflow

![overflow](https://github.com/tailwindlabs/tailwindcss/assets/129918198/dbc94a4c-674d-4bd4-b0fd-71cac0ed978d)

- after overflow

![overflow2](https://github.com/tailwindlabs/tailwindcss/assets/129918198/c9187d2c-aaee-4a4a-b3c5-55e005286a6b)

I need to use overflow and would like to solve this problem. What should I do?

**Comment by marketingBBB** [comment]
https://amzn.to/44WQGlX

**Comment by zetaab** [comment]
@kimmingyu1234756 did you find any solution to this? I do have same issue currently with all devices.
